tracer/test/Datadog.Trace.Security.IntegrationTests/RASP/AspNetCore5Rasp.cs

```diff
@@ -108,6 +106,12 @@ public async Task TryStartApp()
     [Trait("RunOnWindows", "True")]
     public async Task TestRaspRequest(string url, string exploit)
     {
+        AddHeaders(new()
+        {
+            { "Accept-Language", "en_UK" },
```

Comment:
I think this isn't actually a "normal" value, usually it has a format like this: `en-GB`. Does that matter?

Reply:
Not really. I copied that from a waf test to compare. It would only affects to the hash result.